module-db/tests/Helpers.cpp

```diff
@@ -0,0 +1,150 @@
+        const auto postfix = name.substr(name.find_last_of("_") + 1, std::string::npos);
+
+        return {name, prefix, postfix};
+    }
+
+    bool isOnExcludedList(const std::string &name, const std::vector<std::string> &excludedList)
+    {
+        return std::any_of(excludedList.begin(), excludedList.end(), [&name](const auto &entry) {
+            return name.find(entry) != std::string::npos;
+        });
+    }
+
+    std::vector<std::filesystem::path> listFiles(const std::filesystem::path &path,
+                                                 const std::string &prefix,
+                                                 const std::vector<std::string> &excludedList)
+    {
+        std::set<std::pair<int, std::filesystem::path>> orderedFiles;
+        for (const auto &entry : std::filesystem::directory_iterator(path)) {
+            if (!entry.is_directory() && entry.path().has_filename()) {
+                try {
+                    const auto parts       = splitFilename(entry.path().filename().string());
+                    const auto &filePrefix = parts[1];
+                    if (filePrefix == prefix and not isOnExcludedList(parts[0], excludedList)) {
+                        orderedFiles.insert({std::stoi(parts[2]), entry.path()});
+                    }
+                }
+                catch (std::invalid_argument &e) {
+                    printf("Ignoring file: %s", entry.path().c_str());
+                }
+            }
+        }
+
+        std::vector<std::filesystem::path> files;
+        std::for_each(orderedFiles.begin(), orderedFiles.end(), [&](auto item) { files.push_back(item.second); });
+        return files;
+    }
+
+    bool executeOnDb(Database &db, const std::vector<std::string> &statements)
+    {
+        return std::all_of(statements.begin(), statements.end(), [&db](const auto &statement) {
+            return db.execute(statement.c_str());
+        });
+    }
+} // namespace
+
+namespace db::tests
+{
+    std::filesystem::path currentFileLocation()
+    {
+        const std::string path = __FILE__;
+        return std::filesystem::path{path.substr(0, path.rfind('/'))};
+    }
+
+    std::filesystem::path getScriptsPath()
+    {
+        return currentFileLocation() / "../databases/scripts";
+    }
+
+    /// TODO:
+    /// This function is obviously wrong and should not exist in the first place. Unfortunately, due to the
+    /// current directory structure, product specific modules and some unit tests are placed in common directories.
+    /// After proper module-db and service-db split it wont' be needed as every product-specific db unit test will be
+    /// placed in the correct directory.
+    std::filesystem::path getPurePhoneScriptsPath()
+    {
+        return currentFileLocation() / "../../products/PurePhone/services/db/databases/scripts";
+    }
+
+    bool DatabaseScripts::operator()(Database &db)
+    {
+        const std::filesystem::path dbpath = db.getName();
```

Comment:
Not for this PR, but a general remark by the way: I think we could get rid of this unnecessary assignments "padding"...